### PR TITLE
Added Guacamole database to unit test example

### LINKED ISSUE #50: Unit tests do not use in memory database in unit test settings
Running any unit tests requires configuration of MySQL.
Unit test settings should override this and replace MySQL with Sqlite.

python manage.py test --settings=calipsoplus.settings_unittests apprest/tests/commands/

  File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/base/base.py", line 216, in ensure_connection
    self.connect()
  File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/base/base.py", line 194, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 227, in get_new_connection
    return Database.connect(**conn_params)
  File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/MySQLdb/__init__.py", line 86, in Connect
    return Connection(*args, **kwargs)
  File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/MySQLdb/connections.py", line 204, in __init__
    super(Connection, self).__init__(*args, **kwargs2)
_mysql_exceptions.OperationalError: (1698, "Access denied for user 'aidan'@'localhost'")

### PR DESCRIPTION
**Pull request by aicampbell**
close #50

### CODE REVIEW

**Review by dsanchez-cells — CHANGES_REQUESTED** [review]
I agree with redefining the DATABASES variable (because if not you are still stuck with guacamole as MySQL), but you don't need to define two separate databases for unit test, declaring just default should suffice as long as there are no conflicts with the names of the tables (declaring two databases without changes in database router just ends up with the model duplicated in both databases).

In any case, as the guacamole model classes are currently not managed, Django does not create the tables or touch them in any way. This has not been an issue because the guacamole model is not used in the tests at the moment, but we can look into it later on.

Bottom line, for unit tests, better just one database.

**Comment by aicampbell** [comment]
I tried to run the unit tests using a fresh pull from the repository.

I got the following error:
File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/base/base.py", line 216, in ensure_connection
self.connect()
File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/base/base.py", line 194, in connect
self.connection = self.get_new_connection(conn_params)
File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 227, in get_new_connection
return Database.connect(**conn_params)
File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/MySQLdb/init.py", line 86, in Connect
return Connection(*args, **kwargs)
File "/home/aidan/development/calipsoplus-backend/venv/lib/python3.5/site-packages/MySQLdb/connections.py", line 204, in init
super(Connection, self).init(*args, **kwargs2)
_mysql_exceptions.OperationalError: (1698, "Access denied for user 'aidan'@'localhost'")

So something must be using the Guacamole database somewhere. 
Either way, by adding the second database, the unit tests can be run

**Comment by acampsm** [comment]
./manage.py test  --settings=calipsoplus.settings_unittests

with this settings:

DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': 'mydatabase'
    },
    'guacamole': {
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': 'guacamole'
    }
}

**Comment by dsanchez-cells** [comment]
Ok with the changes then, I thought we had it differently.

**Comment by aicampbell** [comment]
It shouldn't matter for too long.
I already have a prototype which removes the Guacamole db completely

**Review by dsanchez-cells — APPROVED** [review]

**Review by acampsm — APPROVED** [review]
